public/.htaccess

```diff
@@ -0,0 +1,93 @@
+### SILVERSTRIPE START ###
+
+# Deny access to templates (but allow from localhost)
+<Files *.ss>
+    Require ip 127.0.0.1
+</Files>
+
+# Deny access to IIS configuration
+<Files web.config>
+    Require all denied
+</Files>
+
+# Deny access to YAML configuration files which might include sensitive information
+<Files ~ "\.ya?ml$">
+    Require all denied
+</Files>
+
+# Route errors to static pages automatically generated by SilverStripe
+#ErrorDocument 404 /assets/error-404.html
+ErrorDocument 404 /public/errors/error-404.html
+ErrorDocument 500 /assets/error-500.html
```

Comment:
These files don't exist

Reply:
Have commented out, leave a note say they don't work (see my other reply above) - I can just remove the commented out ErrorDocument's if you want